tests/frontend/test_tdmprogram.py

```diff
@@ -452,3 +500,157 @@ class DummyCompiler(sf.compilers.compiler.Compiler):
 
     with pytest.raises(sf.program_utils.CircuitError, match=match):
         new_prog = prog.compile(device=spec, compiler=DummyCompiler())
+
+
+## Test for the compilation
+
+import blackbird as bb
```

Comment:
Maybe worth moving the import to the top level of the file?

Reply:
Done.